packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -0,0 +1,178 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+
+import '../../all-components';
+import type { WaferMapDie, WaferMapColorsScale } from '../types';
+import {
+    WaferMapQuadrant,
+    WaferMapOrientation,
+    WaferMapColorsScaleMode
+} from '../types';
+import {
+    HIGHLIGHTEDVALUESETS,
+    WAFERMAPDIESETS,
+    WAFERMAPCOLORSCALESETS
+} from './sets';
+
+interface WaferMapArgs {
+    quadrant: WaferMapQuadrant;
+    orientation: WaferMapOrientation;
+    maxCharacters: number;
+    dieLabelsHidden: boolean;
+    dieLabelsSuffix: string;
+    colorsScaleMode: WaferMapColorsScaleMode;
+    highlightedValues: number[];
+    dies: WaferMapDie[];
+    colorScale: WaferMapColorsScale;
+}
+
+const metadata: Meta<WaferMapArgs> = {
+    title: 'WaferMap',
+    parameters: {
+        docs: {
+            description: {
+                component: 'A WaferMap description'
+            }
+        },
+        actions: {
+            handles: ['click', 'mouseover']
+        }
+    },
+    render: createUserSelectedThemeStory(html`
+        <nimble-wafer-map
+            id="waferMapEl"
+            quadrant="${x => x.quadrant}"
+            orientation="${x => x.orientation}"
+            maxCharacters="${x => x.maxCharacters}"
+            dieLabelsHidden="${x => x.dieLabelsHidden}"
+            dieLabelsSuffix="${x => x.dieLabelsSuffix}"
+            colorsScaleMode="${x => x.colorsScaleMode}"
+            :highlightedValues="${x => x.highlightedValues}"
+            :colorScale="${x => x.colorScale}"
+            :dies=${x => x.dies}
+        >
+        </nimble-wafer-map>
+    `),
+    args: {
+        dies: WAFERMAPDIESETS[0],
+        quadrant: WaferMapQuadrant.bottomLeft,
+        orientation: WaferMapOrientation.left,
+        colorScale: WAFERMAPCOLORSCALESETS[0],
+        maxCharacters: 4,
+        dieLabelsHidden: false,
+        dieLabelsSuffix: '',
+        colorsScaleMode: WaferMapColorsScaleMode.linear,
+        highlightedValues: HIGHLIGHTEDVALUESETS[0]
+    },
+    argTypes: {
```

Comment:
I noticed in the [rendered storybook](https://60e89457a987cf003efc0a5b-vtvohnsyfa.chromatic.com/?path=/story/wafermap--wafer-map) that some of the enum values show an initial selection (quadrant, orientation, colorsScaleMode) but others show "Choose option..." (dies, colorScale, highlightedValues). Since the story always renders with some initial data I believe it would be more accurate to always show one of the items being selected. 

I'm not sure exactly how storybook decides whether to show the "Choose option..." item but could you look into removing it when it makes sense?

Reply:
Yup ... I did quick pass over the documentation - and indeed there is a parameter "defaultValue" that seems to do just that. I adjusted the code and it did seem to work initially, however it seems that I run into this [issue](https://stackoverflow.com/questions/69850139/storybook-argtype-and-defaultvalue-with-a-select-losing-values-when-changing-vie) ... at this point I do not know how to properly address this, so any suggestion is more then welcome :) . 

P.s. I had to go down this way, with mapping the options values, because the set values are arrays ...

Reply:
I did have sync-up with Milan and we ended up implementing a workaround that seems to solve this issue. I submitted it as well